Library/Formula/libutf8proc.rb

```diff
@@ -0,0 +1,25 @@
+class Libutf8proc < Formula
```

Comment:
Done! Sorry; thought the "check what Debian calls it" guideline over rid this principle.